lib/interface_quda.cpp

```diff
@@ -5539,12 +5111,11 @@ void performWuppertalnStep(void *h_out, void *h_in, QudaInvertParam *inv_param,
 
 void performTwoLinkGaussianSmearNStep(void *h_in, QudaQuarkSmearParam *smear_param)
 {
-  if(smear_param->n_steps == 0) return;
+  if (smear_param->n_steps == 0) return;
+  pushProfile(profileGaussianSmear);
+  profileGaussianSmear.TPSTART(QUDA_PROFILE_INIT);
```

Comment:
If `QUDA_PROFILE_INIT` has been removed elsewhere, why is it still included here?

Reply:
While I had intentionally engineered ref counting, so that we can double start the same timer, to avoid any breakages, given you comment here, I'll just go ahead and remove all unnecessary `QUDA_PROFILE_INIT` start and stops in `interface_quda.cpp`.

Reply:
[3963f63](https://github.com/lattice/quda/pull/1384/commits/3963f6329ef8af6d3cc56d1ce6b44859bdf77dbb)